docs/users/interact-with-gnokey.md

```diff
@@ -979,9 +966,30 @@ In practice, this is shorthand for listing packages under `gno.land/p/foo` &
 
 ### Gas parameters
 
-When using `gnokey` to send transactions, you'll need to specify gas parameters:
+Gnokey supports **automatic gas estimation** by default, making transactions 
+easier to send without manually specifying gas parameters:
 
 ```bash
+# Automatic gas estimation (default)
+gnokey maketx call \
+  --pkgpath "gno.land/r/demo/boards" \
+  --func "CreateBoard" \
+  --args "MyBoard" "Board description" \
+  --remote https://rpc.gno.land:443 \
+  --chainid staging \
+  YOUR_KEY_NAME
+```
+
+#### Manual Gas Control (Optional)
+
+For advanced users who want to control gas parameters manually, you can 
+specify:
+
+- `--gas-wanted` - Maximum gas units to consume (or "auto" for estimation)
+- `--gas-fee` - Fee per gas unit (or "auto" for automatic pricing)
```

Comment:
```suggestion
- `--gas-fee` - Amount to pay (or "auto" for automatic pricing)
```
Same comment as above. It is not "per gas unit". The `-gas-fee` is the actual number of coins to be paid.